backend/src/main/resources/static/semantic-hub-openapi.yaml

```diff
@@ -128,6 +128,49 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /models/bulk:
```

Comment:
We might consider to change the endpoint to avoid collisions with the /models/{urn} endpoint, even bulk is not an expected value for {urn}.